### ISSUE TITLE
Can OVPsim run dynamically linked files?

### ISSUE THREAD

**Issue by joy2myself** [riscv-admin/riscv-ovpsim#24]
I found only guidance for running static elf files in both user manuals and test cases. Can OVPsim run dynamically linked files?

**Comment by Imperas** [comment]
You are correct, the github version of riscvOVPsim is intended for running a bare metal statically linked elf file.

Can you explain a little more about what you are trying to do ?

Thx
Lee
